Backend/app/user/user_router.py

```diff
@@ -118,5 +121,18 @@ async def face(user_email: str, face: UploadFile):
     await upload_facial_profile(user_email, face_file)
 
 
+@router.post("/login_face_recognition")
+async def login_face_recognition(user_email: str, face: UploadFile):
+    bytes_image = await face.read()
+    result = await compare_facial_profile(user_email, bytes_image)
+
+    if not "error" in result | result == False:
```

Comment:
También para hacer `error in result` la función `compare_facial_profile` debería funcionar diferente. Ahora mismo lo que hace es tirarte un `HTTPError` directamente. Si queres podes modificar estas funciones para que si hubo un error te tire un diccionario con error como el que da `user_service.update_user`. Si no podes modificarlas para que tiren un `HTTPException` de FastAPI directamente (el que tiran ahora es de la biblioteca requests). Y si no podes usar un `try/catch` y transformar el `HTTPError` en un `HTTPException`.